providers/feature_dism.rb

```diff
@@ -69,3 +68,22 @@ def dism
     locate_sysnative_cmd('dism.exe')
   end
 end
+@@feature_list = nil
+def get_feature_list
+  if @@feature_list.nil?
+    list = {}
+    out = shell_out("#{dism} /online /Get-Features", returns: [0, 42, 127]).stdout
```

Comment:
Nice, spot. There is an entire line missing. Not sure how that made it through testing.